demo/client/backend/fabric-gateway/src/fabricClient.js

```diff
@@ -11,6 +11,8 @@ const { Gateway, FileSystemWallet, X509WalletMixin } = require('fabric-network')
 const fs = require('fs');
 const path = require('path');
 
+const logger = require('fabric-network/lib/logger').getLogger('Transaction');
```

Comment:
Is this logger for the node-sdk patch?

Reply:
good point: at some point i wanted to replace the console.logs with logger but then stopped (or it didn't work; cant' remember): -> deleted it now ...